_posts/2019-07-19-paho-mqtt.md

```diff
@@ -86,17 +86,17 @@ You will see a file containing something like this:
 # The following lines are desirable for IPv6 capable hosts
```

Comment:
Wait, nothing of this is IPv6. Why are we talking about IPv6? I mean, in theory, it could be.

Reply:
It's just part of the file, I can just add that to the general [...] though if you prefer

Reply:
actually that would make significantly more sense wouldn't it

Reply:
So, IPv6 is a very different topic and sadly properly understood by few. Let's stick to IPv4 for the tutorials (unless we do an IPv6 specific one).